tempesta_fw/t/functional/tests/test_cache.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
```

Comment:
Please add haeader comment to ALL THE TESTS describing what and how the particular test verifies.